src/api.go

```diff
@@ -357,3 +358,14 @@ func RoleIdForName(name *C.char) C.int {
 	r := role.FromString(s)
 	return C.int(r)
 }
+
+//export UastReadMemStats
+// UastReadMemStats fills in current memory statistics. This call may affect libuast performance.
```

Comment:
Consider https://github.com/golang/go/wiki/CodeReviewComments#initialisms (UAST)

Reply:
If I understood correctly, C convention is to name it `Uast`. And I cannot change it only on Go side because it will conflict with `//export`.